### PR TITLE
AB#106546 update project - fix issue where data was being overwritten with null values.

### PR DESCRIPTION
**Pull request by jordan-beard**
Addresses bug where data in the project table was being overwritten with empty values.

### CODE REVIEW

File: Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs [hunk 1]
```diff
@@ -71,60 +59,9 @@ public async Task ProjectExists___FullProjectIsUpdated()
 		var updateResult = await legacyProjectController.Patch(updatedProject.Id, updatedProject);
 
 		// Assert
-		DfeAssert.OkObjectResult(updateResult);
-
-		var getResult = await legacyProjectController.Get(updatedProject.Id);
+		(_, var project) = DfeAssert.OkObjectResult(updateResult);	
 
-		(_, var getProject) = DfeAssert.OkObjectResult(getResult);
```
Comment (dneed-nimble) [inline Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs:78]: Is it worth having another test (ProjectExists_FullProjectIsReturnedOnGet or something?) to throw the removed items here that are testing the GET request?

Not a massive amount of value in it but would be a way to repurpose these removed aspects..

**Review by dneed-nimble — APPROVED** [review]

File: Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs (hunk 1 above)
Comment (jordan-beard) [reply Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs:78]: Sure, I'll add that now. Thanks for the comment.

Comment (jordan-beard) [reply Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs:78]: There is LegacyProjectGetTests.cs as well. But there's definitely know harm in adding an extra test here.

Comment (jordan-beard) [reply Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs:78]: Done.

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**